miden-lib/asm/miden/account.masm

```diff
@@ -101,6 +101,37 @@ export.set_item
     # => [R', V]
 end
 
+#! Gets a map item from the account storage. Panics if the index is out of bounds.
+#!
+#! Stack: [index, KEY]
+#! Output: [VALUE]
+#!
+#! - index is the index of the item to get.
+#! - VALUE is the value of the item.
+export.get_map_item
+    push.0 movdn.5 push.0 movdn.5 push.0 movdn.5
+    # => [index, KEY, 0, 0, 0]
+    syscall.get_account_map_item
+    # => [VALUE]
+end
+
+#! Sets a map item in the account storage. Panics if the index is out of bounds.
+#!
+#! Stack: [index, KEY, VALUE]
+#! Output: [OLD_MAP_ROOT, OLD_MAP_VALUE]
+#!
+#! - index is the index of the item to set.
+#! - VALUE is the value to set at KEY.
+#! - OLD_MAP_ROOT is the old map root.
+#! - OLD_MAP_VALUE is the old value at KEY.
+export.set_map_item
+    push.0 movdn.9 push.0 movdn.9 push.0 movdn.9
+    # => [index, KEY, VALUE, 0, 0, 0]
+
+    syscall.set_account_map_item
+    # => [OLD_MAP_ROOT, OLD_MAP_VALUE,]
```

Comment:
**Request**: If we do update `set_account_map_item` to pad the return to the same length as the inputs, this will need something like:

```
movup.8 drop
```

Reply:
Now it should be correct.